### DISCUSSION TITLE
IRC?

### DISCUSSION THREAD

**Discussion by knackstedt** [zen-fs/core#201] [Q&A]
I'm really excited for the new version of zen-fs that's coming out. Is there any IRC chats (discord/slack etc) for this project?

I'm willing to create a Discord server if there's an appetite for it

**Comment by james-pre** [comment]
I don't believe there is a discord server or slack project for zenfs. I would be willing to create one after I finish some of the more pressing issues.

**Comment by knackstedt** [comment]
I can offer myself to make a Discord server for it -- making you and whoever else you specify an admin/team member.
If you want to create it, it's no big deal -- I'm just trying to help out :)

**Comment by james-pre** [comment]
@knackstedt,

I've made a discord server for ZenFS. You can join here: https://zenfs.dev/discord

**Comment by james-pre** [comment]
Please use [zen-fs/discussions](https://github.com/orgs/zen-fs/discussions)!
